pkg/endpointmanager/conntrack.go

```diff
@@ -80,10 +75,10 @@ func RunGC(e *endpoint.Endpoint, isLocal, isIPv6 bool, filter *ctmap.GCFilter) {
 }
 
 // EnableConntrackGC enables the connection tracking garbage collection.
-func EnableConntrackGC(ipv4, ipv6 bool) {
+func EnableConntrackGC(ipv4, ipv6 bool, gcinterval int) {
 	go func() {
```

Comment:
Sorry, I just realized something, add an `if` to check if `gcinterval` is less than 1. If it is, then assign `1` to `gcinterval`, just to prevent user to set up the GC to run every second. Maybe 5? defer to @joestringer

Reply:
Yeah, I think anything less than 5 is pretty unwise.